extensions/amp-inputmask/0.1/test/test-mask-impl.js

```diff
@@ -17,7 +17,9 @@
 import {Mask} from '../mask-impl';
 
 describes.sandboxed('amp-inputmask mask-impl', {}, env => {
-  class FakeElement {}
+  class FakeElement {
+    addEventListener() {}
```

Comment:
Looks like there's another stubbed method in the form of:

```js
FakeElement.prototype.getAttribute = env.sandbox.stub();
```

Is there a reason for them to be different or can they be consolidated as the same type of stubbing?

Reply:
oh good catch, is one better than the other here?

Reply:
`getAttribute` goes through the sandbox since it stubs its return value in a particular test.

Do the same for `addEventListener` for consistency or in case it has to be spied/stubbed later on.